homeassistant/components/binary_sensor/__init__.py

```diff
@@ -89,6 +89,9 @@
 # On means open, Off means closed
 DEVICE_CLASS_WINDOW = "window"
 
+# On means leak detected, Off means no leak
+DEVICE_CLASS_LEAK = "leak"
```

Comment:
You're not allowed to change the entity model in a PR without prior discussing it in the architecture repo as per https://developers.home-assistant.io/docs/entity_index#changing-the-entity-model

Reply:
Also, this is the same as device class moisture?

Reply:
It is moisture class. I missed it. I reverted all changes from binary_sensor.